### PR TITLE
Create onebranch pipeline for win32-wheel

### PR DESCRIPTION
**Pull request by snnn**
For compliance work. This is just the start.

### CODE REVIEW

**Review by natke — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -133,9 +133,6 @@ def read_version():
     return version_str
 
 
-if sys.platform == "win32":
-    load_msvcvar()
```
Comment (wenbingl) [inline setup.py:137]: Why do you need to remove this line?

Comment (snnn) [reply setup.py:137]: This project is cmake-based. You don't need to run cmake and the build commands in VC++'s  environment. You can directly run cmake in a normal cmd.exe Windows. On the other side, when you run this bat, it adds a lot of things to the PATH, not just the compilers. For example, it brings in cmake/python/...

Comment (wenbingl) [reply setup.py:137]: In some environments like Powershell, VC batch file cannot be directly used, that's why this line was added.

Comment (snnn) [reply setup.py:137]: Ok. let me revert it. 
To clarify, you don't need to use the VC batch file. CMake can auto find VC++ paths without the help of the VC batch file

Comment (wenbingl) [reply setup.py:137]: Yes, the cmake as a pip package can find VC++. but there is a bug in pip version cmake sometimes prevent it working with setup.py.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**